.travis.yml

```diff
@@ -25,6 +25,7 @@ before_install:
     -
 install:
     - pip install -r docs/rtd-requirements.txt
+    - pip install -r requirements.txt
```

Comment:
remove this line. The requirements for the CI server are in `docs/rtd-requirements.txt`.